FreeRTOS/FreeRTOS
# Fix AWS IoT Device Defender Demo

Fix Device Defender Demo to retain only the authentication method that works

Description
-----------
Remove the authentication method that uses AWS Lambda function and AWS IoT Custom Authenticator in Device Defender Demo.

Test Steps
-----------
Authentication using certificates and keys works great for Device Defender Demo. But, the above-mentioned method failed to work despite trying every known way.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
